demo/3000-home/next.config.js

```diff
@@ -8,9 +8,6 @@ module.exports = {
         name: 'home_app',
         filename: 'static/chunks/remoteEntry.js',
         remotes: {
-          home: `home_app@http://localhost:3000/_next/static/${
-            isServer ? 'ssr' : 'chunks'
-          }/remoteEntry.js`,
           shop: `shop@http://localhost:3001/_next/static/${
```

Comment:
A global var `home_app@` and `name: 'home_app'` are the same. It would be nice if `home` app determines that `remote.home` is itself and avoid unnecessary staff.

In such a case DevOps engineers can define the one global ENV config amoung all their apps. It will be very convenient not to care about dedicated remote config for every app.

Reply:
I changed the name here to test when global is different than alias. In my case with medusa the global name would be a commit hash. So all resoutions and lookups need to happen off the global, not the alias

Reply:
Ha-ha

Several hours ago I had a similar decision about what to take – `url` or `global` as key for RemoteContainer singleton. And I [chose](https://github.com/module-federation/nextjs-mf/pull/103/files#diff-c0c952ae72a764318951c6a86cbd94840457df635b4e58755a7bb5e8c76251a9R46) `url`.

So I will change it on global var.
I made a small video about what is happening with the client side – https://youtu.be/B0DEr8TQvxI

Reply:
Yeah using global name should be mapped to URL just the same - so both are / should be equally unique. 

Cant have 2 globals pointing to different URLs, global name gives us a good way to search for remotes as containers don't know their own URL on the server, but they know their own name - so we can look up URL for container by searching registry for its "name" 

but users can implement alias for remote as any name they want

so in medusa home: hjui3408h4398oijj320@http:/homeremote.veresion123.com/remoteentry.js